code-studio/src/js/progress.js

```diff
@@ -1,4 +1,8 @@
-/* globals dashboard, appOptions  */
+/* globals appOptions  */
+
+var clientState = require('./clientState');
+var StageProgress = require('./components/progress/stage_progress.jsx');
+var CourseProgress = require('./components/progress/course_progress.jsx');
```

Comment:
Ahh yes, those are from https://github.com/code-dot-org/code-dot-org/pull/7376. Should be fine for them to go in with this PR, but will probably separate them for clarity